Documentation/Signoffs/Resistance_System_Control.md

```diff
@@ -0,0 +1,117 @@
+Team 4 – Mario Kart & Ride Replay Simulator
```

Comment:
I don't see any analysis of the things mentioned in the last comments.

Reply:
What exactly do you mean by analysis?

Reply:
After our discussion earlier and my most recent edit, I believe the issues you mentioned have been resolved.